letsencrypt/cli.py

```diff
@@ -1583,6 +1584,10 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
         "security", "--rsa-key-size", type=int, metavar="N",
         default=flag_default("rsa_key_size"), help=config_help("rsa_key_size"))
     helpful.add(
+        "security", "--ecdsa_curve", default=flag_default("ecdsa_curve"))
+    helpful.add(
+        "security", "--privkey_signature_algorithm", default=flag_default("privkey_signature_algorithm"))
```

Comment:
I would instead call this flag `--key-types` and make it a list of key types. Same comment above about listing options and validating them applies.

The reason to make it a list is that I expect that for people using the autoconfiguration features, they will most likely want to issue both an RSA cert and and ECDSA cert, and configure both of them in their web server, to be offered based on client handshake.

There's some additional groundwork to be laid for having multiple "live" certs per name, which I don't think has to happen in this change. The main thing is that if we're introducing a new flag, I'd like to have its name and semantics be what we want to use for the long term. For now, the validator can reject any list of key types with length greater than one.